### PR TITLE
Added support for various time formats in query api

### PR DESCRIPTION
**Pull request by rohitsngh27**
# Resolves
https://jira.rax.io/browse/CERES-370

# What
Need to add support for various time formats in query params of query api.

# How
Alter the query params to support different time formats.

## How to test
This can be tested via REST call or unit test cases.

### CODE REVIEW

**Comment by rohitsngh27** [comment]
This is first commit to check if I am following the right direction.
Will add the unit test cases later in this PR.

**Review by itzg — COMMENTED** [review]
Looking really good.

**Comment by GeorgeJahad** [comment]
looks fine to me outside of tests and Geoffs comments.

**Comment by rohitsngh27** [comment]
Added unit cases now.

**Review by itzg — CHANGES_REQUESTED** [review]
Looking great, just some minor things.

FYI I queued up a jira issue to add API/controller unit tests
https://jira.rax.io/browse/CERES-372

File: src/test/java/com/rackspace/ceres/app/utils/DateTimeUtilsTest.java [hunk 1]
```diff
@@ -29,57 +27,59 @@ public void getAbsoluteTimeFromRelativeTimeTest_Invalid() {
...
   public void isValidEpochSecondsTest_Invalid() {
-    assertFalse(DateTimeUtils.isValidEpochSeconds("1605094715000"));
+    assertThat(DateTimeUtils.isValidEpochSeconds("1605094715000")).isFalse();
   }
 
   @Test
   public void parseInstantTestWithNull() {
-    assertNotNull(DateTimeUtils.parseInstant(null));
+    assertThat(DateTimeUtils.parseInstant(null)).isNotNull();
   }
```
Comment (itzg) [inline src/test/java/com/rackspace/ceres/app/utils/DateTimeUtilsTest.java]: Can you also assert it's within a second of current time, like you did in `getAbsoluteTimeFromRelativeTimeTest`

**Review by itzg — CHANGES_REQUESTED** [review]
Super close. Just the one unit test.

File: src/test/java/com/rackspace/ceres/app/utils/DateTimeUtilsTest.java (hunk 1 above)
Comment (rohitsngh27) [reply src/test/java/com/rackspace/ceres/app/utils/DateTimeUtilsTest.java]: Sorry...Missed that one

**Review by itzg — APPROVED** [review]
It's so nice to now be able to do a query like:

```
GET http://localhost:8080/api/query?
  tenant=t-1
  &metricName=memory_free
  &tag=os=linux
  &start=1h-ago
```

**Pull request merged**